cloudregister/registercloudguest.py

```diff
@@ -205,7 +202,9 @@ def setup_registry(registration_target):
             )
         )
         return False
-    log.warning('Instance registry setup done, sessions must be restarted !')
+    # This message should be on stdout, see issue#238
+    log.info('Instance registry setup completed, '
+             'all existing shell sessions must be restarted !')
```

Comment:
Nitpick: last time I read the SUSE docs docs there was a mention to avoid words like must. Not sure if that's still the case. Another idea "all existing shell sessions require a restart".